pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
```

Comment:
This description is designed to communicate the following:
- This tool manages groups, and not just a single computer
- You'll need SSH on the remote machines as a pre-req
- It is a remote config/administration tool